tools/gluten-te/cbash.sh

```diff
@@ -34,7 +34,7 @@ CBASH_DOCKER_RUN_ARGS="$CBASH_DOCKER_RUN_ARGS --ulimit core=-1"
 CBASH_DOCKER_RUN_ARGS="$CBASH_DOCKER_RUN_ARGS --security-opt seccomp=unconfined"
 if [ "$MOUNT_MAVEN_CACHE" == "ON" ]
 then
-  CBASH_DOCKER_RUN_ARGS="$CBASH_DOCKER_RUN_ARGS -v $HOME/.m2/repository:/root/.m2/repository"
+  CBASH_DOCKER_RUN_ARGS="$CBASH_DOCKER_RUN_ARGS -v $HOME/.m2/repository:/root/.m2/repository:ro"
 fi
```

Comment:
Instead, we can replace variable `MOUNT_MAVEN_CACHE` with a new variable `MAVEN_CACHE_MODE=HOST/HOST_READONLY/DISABLED`, defaulted to `HOST`. In this case we set `MAVEN_CACHE_MODE=HOST_READONLY`

Reply:
Readonly mode can not work.  I am considering to set MOUNT_MAVEN_CACHE = on for velox-backend only.